packages/account/src/Configs/get-mt5-status-badge-config.tsx

```diff
@@ -12,17 +12,25 @@ const getMT5StatusBadgeConfig = (mt5_account_status: TMT5AccountStatus) => {
             return {
                 text: (
                     <Localize
-                        i18n_default_text='<0>Pending verification</0>'
-                        components={[<Text key={0} weight='bold' size='xxxs' color='var(--status-warning)' />]}
+                        i18n_default_text='<1>In review</1>'
+                        components={[
+                            <Text
+                                key={0}
```

Comment:
@Dhruv-deriv do we need this Text component? There is no key with key 0

Reply:
can we use here Localize inside text component? <Text><Localize/></Text>?

Reply:
done 👍